src/EnergyPlus/HVACUnitaryBypassVAV.hh

```diff
@@ -146,9 +144,6 @@ namespace HVACUnitaryBypassVAV {
         Real64 HeatingSpeedRatio = 1.0;                                      // Fan speed ratio in heating mode
         Real64 CoolingSpeedRatio = 1.0;                                      // Fan speed ratio in cooling mode
         Real64 NoHeatCoolSpeedRatio = 1.0;                                   // Fan speed ratio when no cooling or heating
-        Real64 MaxONOFFCyclesperHourCycling = 4.0;                           // Maximum cycling rate of heat pump [cycles/hr]
-        Real64 HPTimeConstantCycling = 0.0;                                  // Heat pump time constant [s]
-        Real64 FanDelayTimeCycling = 0.0;                                    // Fan delay time, time delay for the HP's fan to
```

Comment:
Remove parent data members now set in child object (again). Going to stop pointing these out because the pattern should be evident. The one place they get a little weird is where the same variables had different defaults depending on the type of parent object.